routing/dht/dht.go

```diff
@@ -322,7 +313,7 @@ func (dht *IpfsDHT) betterPeersToQuery(pmes *pb.Message, p peer.ID, count int) [
 	// == to self? thats bad
 	for _, p := range closer {
 		if p == dht.self {
-			log.Debug("Attempted to return self! this shouldnt happen...")
+			log.Error("Attempted to return self! this shouldnt happen...")
```

Comment:
`log.Error` is for the user, and they pop out when running the daemon (which right now is beautifully silent). this should stay at `log.Debug`.